tests/comdb2sys.test/testapihistory.expected

```diff
@@ -0,0 +1 @@
+(host='m0.comdb2', api_driver_name='open_cdb2api', api_driver_version='latest')
```

Comment:
comdb2sys failed! I think host is most likely 'c1' here? we spawn a client machine as c1, and 3 servers as m1, m2 and m3.

Reply:
Actually I think it would be better if this host column is generated from the `hostname` command, so the test would work everywhere.